specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json

```diff
@@ -1868,6 +1907,43 @@
         }
       }
     },
+    "Tags": {
+      "description": "A tag resource.",
```

Comment:
Please put in more detailed descriptions. This is easily confused with "Tag".

Reply:
You might even consider renaming this to "TagResource".

Reply:
I have updated the description. Please take a look if the changes are fine.